plat/rockchip/common/plat_pm.c

```diff
@@ -412,6 +412,7 @@ const plat_psci_ops_t plat_rockchip_psci_pm_ops = {
 	.pwr_domain_suspend = rockchip_pwr_domain_suspend,
 	.pwr_domain_on_finish = rockchip_pwr_domain_on_finish,
 	.pwr_domain_suspend_finish = rockchip_pwr_domain_suspend_finish,
+	.pwr_domain_pwr_down_wfi = rockchip_pd_pwr_down_wfi,
```

Comment:
@TonyXie06 
This function is missing for rk3368 and rk3399 - checks failed, e.g. for rk3368:

/debug/bl31/plat_pm.d -MT build/rk3368/debug/bl31/plat_pm.o -MP -c plat/rockchip/common/plat_pm.c -o build/rk3368/debug/bl31/plat_pm.o
plat/rockchip/common/plat_pm.c:406:29: error: 'rockchip_pd_pwr_down_wfi' undeclared here (not in a function)
  .pwr_domain_pwr_down_wfi = rockchip_pd_pwr_down_wfi,
                             ^
make: *** [build/rk3368/debug/bl31/plat_pm.o] Error 1